src/components/ArticleView/ArticlePreviewCard.jsx

```diff
@@ -0,0 +1,19 @@
+const ArticlePreviewCard = (article) => {
+  return (
+    <div class="article-preview-container" key={article.article_id}>
+      <section className="head-content">
+      <h2 >
+        {article.title}      
+      </h2>
+      <h3>by {article.author} in {article.topic}</h3>
+      </section>
+      <section className="foot-content">
+      <p>comments {article.comment_count}</p>
```

Comment:
Good work so far - see this article for more interesting ways of displaying information: https://www.refactoringui.com/previews/labels-are-a-last-resort